spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/ReleaseController.java

```diff
@@ -142,6 +143,13 @@ public ReleaseControllerLinksResource resourceLinks() {
 		return this.manifestResourceAssembler.toModel(this.releaseService.manifest(name, version));
 	}
 
+	@RequestMapping(path = "/scale/{name}", method = RequestMethod.POST)
+	@ResponseStatus(HttpStatus.CREATED)
+	public EntityModel<Release> scale(@PathVariable("name") String name, @RequestBody ScaleRequest scaleRequest) {
```

Comment:
Do you think it would be better if we just have the `ScaleRequest` as the Request body and remove the explicit name from the path (given the ScaleRequest already has the name in it)?

Reply:
Not sure, I followed same conventions where release name is always coming from a request path.